README.md

```diff
@@ -66,6 +66,13 @@ $ csc node get-id --endpoint tcp://127.0.0.1:10000
 CSINode
 ```
 
+## Running Kubernetes End To End tests on the NFS Driver
+1) Stand up a local cluster `ALLOW_PRIVILEGED=1 hack/local-up-cluster.sh`
+2) Build the nfs driver by running `make`
+3) Create NFS Driver Image, where the image tag would be whatever that is required by your YAML deployment files        `docker build -t quay.io/k8scsi/nfsplugin:v1.0.0 .`
+4) Run E2E Tests using the following command: `go test -v ./cmd/nfsplugin/ -ginkgo.v -ginkgo.progress --kubeconfig=/var/run/kubernetes/admin.kubeconfig -timeout=0`
```

Comment:
How does someone run the e2e tests with their own private images?

Reply:
you would need to add the required YAML files for that private image in [deploy/kubernetes](https://github.com/kubernetes-csi/csi-driver-nfs/tree/master/deploy/kubernetes), as those are the files that the e2e test uses to deploy the driver. I'll add that to the documentation.

Reply:
I don't see the updated documentation pointing to which yamls need to be changed

Reply:
should we build local image on each node? since the image is not publish to quay.io, if the plugin will run on each node, then should build or import the image to all nodes?

Reply:
just pushed the updated docs